assets/stylesheets/shared/_colors.scss

```diff
@@ -1,74 +1,77 @@
+/**
+ * Colors
+ *
+ * Color variables based on the design handbook
+ * See wordpress.com/design-handbook/colors/ for more info.
+ */
+
 // Primary Accent (Blues)
 $blue-wordpress:         #0087be;
 $blue-light:             #78dcfa;
 $blue-medium:            #00aadc;
 $blue-dark:              #005082;
 
-// Jetpack Green
-$green-jetpack:          #8cc258;
-
 // Grays
 $gray:                   #87a6bc;
 $gray-light:             lighten( $gray, 33% ); //#f3f6f8
+$gray-lighten-30:        lighten( $gray, 30% ); //#e9eff3
+$gray-lighten-20:        lighten( $gray, 20% ); //#c8d7e1
+$gray-lighten-10:        lighten( $gray, 10% ); //#a8bece
+$gray-darken-10:         darken( $gray, 10% ); //#668eaa
+$gray-darken-20:         darken( $gray, 20% ); //#4f748e
+$gray-darken-30:         darken( $gray, 30% ); //#3d596d
```

Comment:
> reduce color variations

It's not a problem we have, or that we intended to "fix" however. We purposefully chose this approach exactly for its flexibility, yet still grounded in the variables.

It's a discussion we could have, but as far as I'm aware it didn't emerge so far as a problem, as such I don't think we should try to fix it. Maybe it's just because the discussion didn't happen yet, but... I wouldn't use the theming PR to change one of the fundamentals of our color approach.